app/config/urls/root.py

```diff
@@ -58,6 +59,12 @@ def handler500(request):
         name="django.contrib.sitemaps.views.sitemap",
     ),
     path(settings.ADMIN_URL, admin.site.urls),
+    re_path(
+        r"accounts/two_factor/setup/?$",
+        TwoFactorSetup.as_view(),
+        name="two-factor-setup",
+    ),
```

Comment:
This doesn't need to be a `re_path`.

```suggestion
    path(
        "accounts/two_factor/setup/",
        TwoFactorSetup.as_view(),
        name="two-factor-setup",
    ),
```

Reply:
Somehow the partial path turns out to be necessary. I'm not sure why, yet, but if I just use path it goes to the original setup view and not the custom one, even thought this url is defined before I include the allauth-2fa urls.